scripts/metalk8s_cli/mixins/salt.py

```diff
@@ -0,0 +1,119 @@
+import json
+import shlex
+import subprocess
+
+import six
+
+from metalk8s_cli.exceptions import CommandError, CommandInitError
+
+
+class SaltCommandMixin(object):
+    def __init__(self, args):
+        super(SaltCommandMixin, self).__init__(args)
+
+    def run_salt_minion(
+        self, command, local=False, outputter=None, color=True,
+        saltenv=None, pillar=None,
+    ):
+        full_cmd = ['salt-call']
```

Comment:
`--retcode-passthrough` I think even if we don't want to fail I think it's good to have the return code

Reply:
Good point, will do (those helpers are really a draft, so thanks for the help!)